src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -209,6 +209,22 @@
                 </StackPanel>
             </GroupBox>
 
+            <GroupBox x:Uid="InlineDiagnosticsGroupBox"
+                      Header="{x:Static local:AdvancedOptionPageStrings.Option_Inline_Diagnostics_experimental}">
+                <StackPanel>
+                    <CheckBox x:Name="DisplayInlineDiagnostics"
+                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Display_inline_diagnostics}"/>
+                    <StackPanel Margin="15, 0, 0, 0">
+                        <RadioButton GroupName="Inline_diagnostic_location"
+                                     x:Name="hook_to_end_of_text"
+                                     Content="{x:Static local:AdvancedOptionPageStrings.Option_Hook_to_end_of_text}"/>
+                        <RadioButton GroupName="Inline_diagnostic_location"
+                                     x:Name="hook_to_end_of_window"
+                                     Content="{x:Static local:AdvancedOptionPageStrings.Option_Hook_to_end_of_window}"/>
+                    </StackPanel>
```

Comment:
we already have an analysis section, so i would just put this in that section.

Reply:
Moved